add-convo-record-for-every-convo.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from sqlalchemy import create_engine, insert
+from sqlalchemy.orm import sessionmaker
+from secure_message.repository.database import SecureMessage, Conversation
+from config import Config
+
+engine = create_engine(Config.SQLALCHEMY_DATABASE_URI)
+Session = sessionmaker(bind=engine)
+db = Session()
+logging.basicConfig()
+logging.getLogger('sqlalchemy.engine').setLevel(logging.INFO)
```

Comment:
The logs now aren't very useful, maybe add some context to say what's being done or remove?